src/components/fieldText/fieldText.tsx

```diff
@@ -60,10 +64,23 @@ export const FieldText: FC<FieldTextProps> = forwardRef(
       hasDoubleMessage = false,
       type = 'text',
       displayError = true,
+      collapsible = false,
+      loading = false,
       ...rest
     },
-    ref: ForwardedRef<HTMLInputElement>,
+    ref,
   ): ReactElement => {
+    const internalRef = useRef<HTMLInputElement>(null);
+    const [focused, setFocused] = useState(false);
+    const onFocus = () => setFocused(true);
+    const onBlur = () => setFocused(false);
+
+    useImperativeHandle(ref, () => ({
+      focus: () => {
+        internalRef.current?.focus();
+      },
+    }));
```

Comment:
Two questions regarding this approach:
1. After this changes it will not be possible to fully control DOM input via provided ref. As it is a common component from library I suppose it should provide more control for parent code e.g. when it is used as an input for more complex controls (like autocompletes, etc.). Can we still leave the possibility for external code to control the input DOM element?
2. Looks like this hook will be triggered each render, is it expected? Can we limit the number of it calls (e.g. do it only on mount)?